app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,7 @@
             android:name=".ListenerActivity"
             android:exported="true" />
         <activity
-            android:name=".MainActivity"
+            android:name=".InfoActivity"
```

Comment:
Wouldn't this change the main screen? Because of this code.
```
<intent-filter>
                <action android:name="android.intent.action.MAIN" />
                <category android:name="android.intent.category.LAUNCHER" />
            </intent-filter>
```

Reply:
You're right. The app's entry point is set incorrectly. It's probably code for debug.

Reply:
Yes, as @kuro11pow2  said this code is for debugging.